src/axom/sidre/interface/c_fortran/wrapfsidre.F

```diff
@@ -1865,277 +1865,277 @@ function c_view_get_owning_group(self, SHT_crv) &
+            import :: SIDRE_SHROUD_view_capsule
             implicit none
-            type(SHROUD_view_capsule), intent(IN) :: self
+            type(SIDRE_SHROUD_view_capsule), intent(IN) :: self
             integer(C_INT) :: SHT_rv
         end function c_view_get_num_dimensions
 
         function c_view_get_shape(self, ndims, shape) &
                 result(SHT_rv) &
                 bind(C, name="SIDRE_View_get_shape")
             use iso_c_binding, only : C_INT, C_INT64_T
-            import :: SHROUD_view_capsule
+            import :: SIDRE_SHROUD_view_capsule
             implicit none
-            type(SHROUD_view_capsule), intent(IN) :: self
+            type(SIDRE_SHROUD_view_capsule), intent(IN) :: self
             integer(C_INT), value, intent(IN) :: ndims
             integer(C_INT64_T), intent(OUT) :: shape(*)
             integer(C_INT) :: SHT_rv
         end function c_view_get_shape
 
         subroutine c_view_allocate_simple(self) &
                 bind(C, name="SIDRE_View_allocate_simple")
-            import :: SHROUD_view_capsule
+            import :: SIDRE_SHROUD_view_capsule
             implicit none
-            type(SHROUD_view_capsule), intent(IN) :: self
+            type(SIDRE_SHROUD_view_capsule), intent(IN) :: self
         end subroutine c_view_allocate_simple
 
         subroutine c_view_allocate_from_type(self, type, num_elems) &
                 bind(C, name="SIDRE_View_allocate_from_type")
             use iso_c_binding, only : C_INT, C_INT64_T
-            import :: SHROUD_view_capsule
+            import :: SIDRE_SHROUD_view_capsule
             implicit none
-            type(SHROUD_view_capsule), intent(IN) :: self
+            type(SIDRE_SHROUD_view_capsule), intent(IN) :: self
             integer(C_INT), value, intent(IN) :: type
             integer(C_INT64_T), value, intent(IN) :: num_elems
         end subroutine c_view_allocate_from_type
 
         subroutine c_view_reallocate(self, num_elems) &
                 bind(C, name="SIDRE_View_reallocate")
             use iso_c_binding, only : C_INT64_T
-            import :: SHROUD_view_capsule
+            import :: SIDRE_SHROUD_view_capsule
             implicit none
-            type(SHROUD_view_capsule), intent(IN) :: self
+            type(SIDRE_SHROUD_view_capsule), intent(IN) :: self
             integer(C_INT64_T), value, intent(IN) :: num_elems
         end subroutine c_view_reallocate
 
         subroutine c_view_attach_buffer_only(self, buff) &
                 bind(C, name="SIDRE_View_attach_buffer_only")
-            import :: SHROUD_buffer_capsule, SHROUD_view_capsule
+            import :: SIDRE_SHROUD_buffer_capsule, SIDRE_SHROUD_view_capsule
             implicit none
-            type(SHROUD_view_capsule), intent(IN) :: self
-            type(SHROUD_buffer_capsule), intent(IN) :: buff
+            type(SIDRE_SHROUD_view_capsule), intent(IN) :: self
+            type(SIDRE_SHROUD_buffer_capsule), intent(INOUT) :: buff
```

Comment:
Same question for the next few functions... were these expected to change form `IN` to `INOUT` ?

Reply:
Yes. See above.